src/content/tests/checkText.test.js

```diff
@@ -1,5 +1,50 @@
-describe('Test', () => {
-  test('testing jest', () => {
-    expect(true).toEqual(true);
+import checkText from '../functions/checkText';
+import {
+  SELECTOR_ONLINE_TWEET,
+  OPT_OUT_API_URL
+} from '../constants';
+
+const fetchMock = require("fetch-mock");
+fetchMock.mock(OPT_OUT_API_URL, 200);
```

Comment:
Ideally we mock the successful response body and the different possible sucessful response bodies.

We should also mock an error status code.

Reply:
Done